### ISSUE TITLE
vendors css is not created since v2.30.0-canary.0

### ISSUE THREAD

**Issue by mawwerik** [carbon-design-system/carbon-components-vue#997]
## Detailed description

When I run ```yarn build``` I would expect that the build process (vue-cli-service build) creates 4 files:
```
dist/js/chunk-vendors.xxx.js
dist/js/app.xxx.js
dist/css/chunk-vendors.xxx.css
dist/css/app.xxx.css
```
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/8342041/91957288-1cfab300-ed06-11ea-933b-6f0eac1344c7.png">

This worked fine up until ```@carbon/vue 2.29.0```.

**Since ```@carbon/vue 2.30.0-canary.0``` the ```dist/css/chunk-vendors.xxx.css``` is not created anymore.**
<img width="1035" alt="image" src="https://user-images.githubusercontent.com/8342041/91957382-3ac81800-ed06-11ea-87b7-b2a1b4a964eb.png">

## Steps to reproduce the issue

1. In ```package.json``` in dependencies section change the version number of ```@carbon/vue``` to ```2.30.0-canary.0```
<img width="296" alt="image" src="https://user-images.githubusercontent.com/8342041/91958054-246e8c00-ed07-11ea-97f2-dfc4d9c1acba.png">

2. Run ```yarn install```
3. Run ```yarn build```

**Comment by lee-chase** [comment]
I am unable to replicate this behaviour by simply changing between versions of @carbon/vue I always get the latter.

Are you able to create a simple project on github demonstrating this issue? Perhaps with before and after branches.

**Comment by mawwerik** [comment]
Hi @lee-chase,

Please remove the ^ sign in front of the version number of @carbon/vue. Refer with the exact version numbers:
1. "@carbon/vue": "2.30.0-canary.0"
2. "@carbon/vue": "2.29.0"

**Comment by lee-chase** [comment]
When I build with 2.29.0 I get the following which would appear to place the contents of the vendors css in the app css file

![image](https://user-images.githubusercontent.com/15086604/92382032-d5a86400-f103-11ea-83c9-d54e3f89c80e.png)

**Comment by mawwerik** [comment]
@lee-chase 2.29.0 is fine. Pls run with version 2.30.0-canary.0.

**Comment by hammertoe** [comment]
I am seeing the same with project on IBM internal Github: https://github.ibm.com/Matthew-Hamilton1/vcpi-tool

```bash
matt@Matts-MBP dist % find . -ls
9906263        0 drwxr-xr-x    6 matt             staff                 192 11 Sep 00:02 .
9906266        8 -rw-r--r--    1 matt             staff                 726 11 Sep 00:02 ./index.html
9906264        0 drwxr-xr-x    3 matt             staff                  96 11 Sep 00:02 ./css
9906268        8 -rw-r--r--    1 matt             staff                 145 11 Sep 00:02 ./css/app.c174350d.css
9906265        0 drwxr-xr-x    6 matt             staff                 192 11 Sep 00:02 ./js
9906269       24 -rw-r--r--    1 matt             staff                9268 11 Sep 00:02 ./js/app.1e4513e3.js
9906272      920 -rw-r--r--    1 matt             staff              470074 11 Sep 00:02 ./js/chunk-vendors.cac694d5.js
9906271     3704 -rw-r--r--    1 matt             staff             1893347 11 Sep 00:02 ./js/chunk-vendors.cac694d5.js.map
9906270       56 -rw-r--r--    1 matt             staff               26112 11 Sep 00:02 ./js/app.1e4513e3.js.map
9906267        8 -rw-r--r--    1 matt             staff                1077 11 Sep 00:02 ./bee_128.png
```

```bash
matt@Matts-MBP vcpi-tool % grep carbon package.json 
    "@carbon/vue": "^2.33.0",
```

**Comment by lee-chase** [comment]
Thanks @hammertoe. Playing with your repository appears to suggest that when carbon moved from v10.12.0 to 10.13.0-rc.0 the vendors css file stops getting generated.

This does 
```
yarn add carbon-components@10.12.0
```

This does not
```
yarn add carbon-components@10.13.0
```

I have not been able to determine why, but the carbon-components css build is outside of my control.

**Comment by lee-chase** [comment]
OK @hammertoe and @mawwerik I now know what is happening.

This https://github.com/carbon-design-system/carbon/pull/5987/files added sideEffects to carbon-components package.json. It results in vender css files not being created.

Submitted this issue https://github.com/carbon-design-system/carbon/issues/6824

**Comment by lee-chase** [comment]
Looks like this will be resolved with carbon-components 10.20.0

**Comment by lee-chase** [comment]
Resolved in 10.20.0 of carbon-components.
